Source/Core/Core/AchievementManager.cpp

```diff
@@ -897,7 +906,8 @@ AchievementManager::ResponseType AchievementManager::Request(
   }
   else
   {
-    WARN_LOG_FMT(ACHIEVEMENTS, "RetroAchievements connection failed.");
+    WARN_LOG_FMT(ACHIEVEMENTS, "RetroAchievements connection failed. \nURL: {} \npost_data: {}",
+                 api_request.url, api_request.post_data);
```

Comment:
I believe I mentioned this before but please check the post_data for null here...